src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # BinSkim Release History
 
+## Unreleased
+
+* BUGFIX: Fix `MultithreadedAnalyzeCommandBase` artifacts generation and enforcing JSON properties ordering. [#555](https://github.com/microsoft/binskim/pull/555)
```

Comment:
>MultithreadedAnalyzeCommandBase [](http://example.com/codeflow?start=15&length=31)

I'd document this isn't a BinSkim fix, it's an SDK update. 

Update SARIF-SDK in order to fix `blah blah